CONTRIBUTING.md

```
@@ -2,27 +2,114 @@
 
 Welcome to Orcasound! We're thrilled you're interested in contributing to the site.
 
-### Code of Conduct
+- [Code of Conduct](#code-of-conduct)
+- [Reporting Bugs](#bugs)
```

Markdownlint

Link fragments should be valid

(MD051, link-fragments)